man/fi_mrail.7.md

```diff
@@ -80,6 +80,12 @@ The ofi_mrail provider checks for the following environment variables.
 *FI_OFI_MRAIL_ADDR_STRC*
 : Comma delimited list of individual rail addresses in FI_ADDR_STR format.
 
+*FI_OFI_MRAIL_STRIPING_THRESHOLD*
+: Minimal message size to allow striping (default: 16385).
```

Comment:
We should find a way to align this with the values used by the underlying providers, or remove it in favor of just using those values.  Otherwise, the admin has to configure an array of values that all align to get the best performance, which is never going to happen.

Reply:
We could query the providers for some properties (like eager buffer size, etc) at some point and derive a value from that at the default. However, being able to override the default can still be useful.

Reply:
The value is already assuming that a specific multirail algorithm is in use.  That is, we're striping a single message across the rails, versus, for example, sending each message on a separate rail, or using different rails for different destinations.

We may need to document all of these unsupported or subject to change, so users don't get too comfortable with them until we have a final version of how to expose everything.  Each algorithm will have its own set of controls.

Keep in mind that I hate environment variables as a general rule.  :)  It's much better for things to just work well.